docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,153 @@
+-   (1-2 points) Add recommended section (only if 1 more more apps on the system are suggested). Since we've just updated the settings list of recommended apps on the system (in prior bullet point), use this settings list - if any system app we discover also appears on that list, it goes in the recommended section.
+-   (1 point) Sort recommended section - selected ones at top alphabetically, then the reset alphabetically
+-   (1 point) Sort "others" section - selected ones at top alphabetically, then the reset alphabetically
+-   (1-2 points) Update sort when something is checked or unchecked. The list dynamically updates when a user is on it.
+-   (1-2 points) Ensure search works, shows one section, is alphabetically sorted
+-   (2-5 points) Ensure accessibility is up to date
+-   (1 point) Add metrics for how many suggested (and non-suggested) apps are excluded\
+
+### Day 2
+
+These are important components that we are committed to delivering, but are outside the scope of this epic. They are items that will be done as soon as the epic is closed, but do not block the feature on shipping.
+
+-   Engineering metrics
+    -   Metrics will be created for a successful deep linking, as well as an erroneous deep linking (the app cannot figure out how to resolve the deep link).
+    -   Add performance metrics around load time of the Excluded Apps screen.
+    -   Create a metric for when the daily job was successful, and another one for when it fails.
+    -   A metric should be recorded when the notification is sent
+
+Other options considered
+------------------------
+
+Design and research went through several iterations, so the [designs](https://www.figma.com/file/UZYzma7hlcfE5ke3z8jGbN/App-exclusions-suggestions?type=design&node-id=196-6366&mode=design&t=RL1hdfBQLMS1rKVa-0) were set before this tech spec was written. (See [completed tickets in the epic](https://mozilla-hub.atlassian.net/browse/VPN-4412) for more background.)
+
+Other options considered included dynamically serving the suggested lists from addons or our repo or Guardian, each of which added complexity for hosting/validating. With the rarity of the list being updated (and lack of time-sensitivity when it does), this draft bundles the lists with the app. Hosting the list in Guardian was difficult (which is a private repo, so the list would need to be hosted elsewhere to allow public suggestions).
+
+### Other options considered for suggested app files
+
+The goals for the file format:
+-   Easy to read by humans
+-   Easy for humans to add to, no matter their level of technical proficiency
+-   Easy for us to parse in the app
+-   Allowing comments in the file. While we still could structure a text file to allow comments, this became less important when the plan became "one file per platform, one app per line".
```

Comment:
IIUC the plan has gone back to one file for all platforms.

Reply:
Thanks, good catch. Fixed.